### ISSUE TITLE
Contribution guide is out of date

### ISSUE THREAD

**Issue by nickaj** [firedrakeproject/firedrake#938]
The contribution guide states:
```
Install a Git pre-commit hook automatically checking for tab and whitespace errors before committing and also calls flake8 on your changed files. In the .git/hooks directory of your local Git repository, run the following:

git config --local core.whitespace "space-before-tab, tab-in-indent, trailing-space, tabwidth=4"
wget https://gist.github.com/kynan/d233073b66e860c41484/raw/pre-commit
chmod +x pre-commit
```
This is bad. Current versions of flake8 (3.0+) don't include the flake8.hooks module so, if you follow the guide and install the hook, you get an error related to importing flake8.hooks when calling git commit.

Either a new hook is required, or the contribution guide needs updated.

**Comment by miklos1** [comment]
We have a contribution guide?

**Comment by wence-** [comment]
Florian... just remove that file completely

**Comment by dham** [comment]
Fixed by  06f498d
